.codesandbox/ci.json

```diff
@@ -4,5 +4,5 @@
     "carbon-quickstart-typescript-6sjfuz"
   ],
   "buildCommand": "precompile",
-  "node": "16"
+  "node": "18"
```

Comment:
**note**: Codesandbox CI currently doesn't support Node 20, [Node 18 is the latest version it supports](https://codesandbox.io/docs/learn/ci#configuration-format)

Reply:
Yeah they're usually a bit behind on updating (understandably)